server/container_stats.go

```diff
@@ -37,10 +37,10 @@ func (s *Server) ContainerStats(ctx context.Context, req *pb.ContainerStatsReque
 		recordOperation(operation, time.Now())
 		recordError(operation, err)
 	}()
-
-	container := s.GetContainer(req.ContainerId)
-	if container == nil {
-		return nil, fmt.Errorf("invalid container")
+	logrus.Debugf("ContainerStatsRequest %+v", req)
```

Comment:
I think this is not needed since we automatically log every RPC in:
https://github.com/cri-o/cri-o/blob/30f477fa1d5014e8afd175d4fa84cae80d3521fe/internal/pkg/log/interceptors.go#L57